src/test/kotlin/no/fdk/concept_catalog/contract/GetConceptRevisions.kt

```diff
@@ -0,0 +1,96 @@
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.fdk.concept_catalog.configuration.JacksonConfigurer
+import no.fdk.concept_catalog.model.Begrep
+import no.fdk.concept_catalog.utils.*
+import no.fdk.concept_catalog.utils.jwk.Access
+import no.fdk.concept_catalog.utils.jwk.JwtToken
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.test.context.ContextConfiguration
+import kotlin.test.assertEquals
+
+private val mapper = JacksonConfigurer().objectMapper()
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SpringBootTest(
+    properties = ["spring.profiles.active=contract-test"],
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+@ContextConfiguration(initializers = [ApiTestContext.Initializer::class])
+@Tag("contract")
+class GetConceptRevisions : ApiTestContext() {
+
+    @Test
+    fun `Unauthorized when access token is not included`() {
+        val rsp = authorizedRequest("/begreper/${BEGREP_0.id}/revisions", port, null, null, HttpMethod.GET)
+
+        assertEquals(HttpStatus.UNAUTHORIZED.value(), rsp["status"])
+    }
+
+    @Test
+    fun `Forbidden for wrong orgnr`() {
+        val rsp = authorizedRequest(
+            "/begreper/${BEGREP_WRONG_ORG.id}/revisions",
+            port,
+            null,
+            JwtToken(Access.ORG_READ).toString(),
+            HttpMethod.GET
+        )
+
+        assertEquals(HttpStatus.FORBIDDEN.value(), rsp["status"])
+    }
+
+    @Test
+    fun `Not found`() {
+        val rsp = authorizedRequest(
+            "/begreper/not-found/revisions",
+            port,
+            null,
+            JwtToken(Access.ORG_READ).toString(),
+            HttpMethod.GET
+        )
+
+        assertEquals(HttpStatus.NOT_FOUND.value(), rsp["status"])
+    }
+
+    @Test
+    fun `Ok for read access`() {
+        val rsp = authorizedRequest(
+            "/begreper/${BEGREP_0.id}/revisions",
+            port,
+            null,
+            JwtToken(Access.ORG_READ).toString(),
+            HttpMethod.GET
+        )
+
+        assertEquals(HttpStatus.OK.value(), rsp["status"])
+
+        val result: List<Begrep> = mapper.readValue(rsp["body"] as String)
+        assertEquals(2, result.size)
+        assertEquals(BEGREP_0, result[0])
+        assertEquals(BEGREP_0_OLD, result[1])
+    }
+
+    @Test
+    fun `Ok for write access`() {
+        val rsp = authorizedRequest(
+            "/begreper/${BEGREP_0.id}/revisions",
+            port,
+            null,
+            JwtToken(Access.ORG_WRITE).toString(),
+            HttpMethod.GET
+        )
+
+        assertEquals(HttpStatus.OK.value(), rsp["status"])
+
+        val result: List<Begrep> = mapper.readValue(rsp["body"] as String)
+        assertEquals(BEGREP_0, result[0])
```

Comment:
Sikkert greit å bare ta det med her og
```suggestion
        assertEquals(2, result.size)
        assertEquals(BEGREP_0, result[0])
        assertEquals(BEGREP_0_OLD, result[1])
```